Source/Core/Core/BootManager.cpp

```diff
@@ -392,6 +392,15 @@ bool BootCore(const std::string& _rFilename)
   SConfig::GetInstance().m_SYSCONF->SetData("IPL.PGS", StartUp.bProgressive);
   SConfig::GetInstance().m_SYSCONF->SetData("IPL.E60", StartUp.bPAL60);
 
+  if (StartUp.bWii)
+  {
+    // Disable WiiConnect24's standby mode. If it is enabled, it prevents us from receiving
+    // shutdown commands in the State Transition Manager (STM).
+    // TODO: remove this if and once Dolphin supports WC24 standby mode.
+    SConfig::GetInstance().m_SYSCONF->SetData("IPL.IDL", 0x00);
```

Comment:
It's unlikely to have corrupted the sysconf file. it would mostly just cause dolphin to crash.